config/ols-client.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: ols-client
+spec:
+  template:
+    metadata:
+      name: ols-client
+    spec:
+      restartPolicy: OnFailure
+      containers:
+      - name: ols-client
+        image: quay.io/brancz/krp-curl:v0.0.2
+        command:
+        - /bin/sh
+        - -c
+        - 'curl -v -s -k -H "Authorization: Bearer `cat /service-account/token`" https://ols-proxy.ols.svc:8443/docs'
```

Comment:
it's not obvious to me what this is doing or why we need it

Reply:
This is just to demonstrate an example on how some client can make requests.

Reply:
cool, got it.  maybe name it "ols-client-test" or "ols-client-sample" just to make it clear it's not really part of the OLS stack